odl/discr/lp_discr.py

```diff
@@ -175,6 +175,11 @@ def max_pt(self):
         return self.partition.max_pt
 
     @property
+    def mid_pt(self):
```

Comment:
We have `partition.midpoint`, we need to standardize this one, i prefer this notation here.